### PR TITLE
fix(design-tokens): update some blues, reds, and focus shadow

### PR DESCRIPTION
**Pull request by TheSisb**
What changed:
```
Console:
    Aliases
        blue-60 —> #027BC8
        blue-90 —> #032D62
        red-60 —> #E22525
        red-90 —> #600101
    Tokens (border + background)
        primary-dark --> blue-80
        primary-darker —> blue-90
        destructive-dark --> red-80
        destructive-darker —> red-90

SG:
    Aliases
        red-90 —> #480500
    Tokens (border + background)
        primary-dark --> blue-70
        primary-darker —> blue-80
        destructive --> red-50
        destructive-darker —> red-90

Both SG and console:
    Focus outline should be 80% opacity on the primary color
        Console: rgba(blue-60, 0.8)
        SG: rgba(blue-50, 0.8)
```

### CODE REVIEW

**Comment by serifluous** [comment]
Fixes:

SG:
    Tokens (border + background)
        primary-dark --> blue-80
        primary-darker —> blue-90
        destructive-dark --> red-80
        destructive-darker —> red-90

        color-text-link-dark --> blue-80
        color-text-link-darker --> blue-90
        color-text-link-destructive-dark --> red-80
        color-text-linke-destructive-darker --> red-90

sorry 😰

**Comment by TheSisb** [comment]
More changes because buttons looked weird:

```

    Focus outline should be 80% opacity on the primary color (This should be tokenized if it isn't already, and it seems like it may need to pull in the token on the button styles).
        Console: rgba(2, 123, 200, 0.8)
        SG: rgba(51, 104, 250, 0.8)
    Primary button
        On hover should use borderColors.colorBorderPrimaryDarker, backgroundColors.colorBackgroundPrimaryDarker
        On focus should use borderColors.colorBorderPrimaryDarker
        When active should use backgroundColors.colorBackgroundPrimaryDark
    Secondary button
        On hover should use textColors.colorTextLinkDarker
        On focus should use textColors.colorTextLinkDarker, borderColors.colorBorderPrimaryDarker
        When disabled should use borderColors.colorBorderPrimaryLight

```

File: packages/paste-design-tokens/tokens/aliases/color-palette.yml [hunk 1]
```diff
@@ -5,12 +5,24 @@ aliases:
...
+  palette-blue-60-transparent-10: "rgba(2, 123, 200, 0.1)"
+  palette-blue-60-transparent-20: "rgba(2, 123, 200, 0.2)"
+  palette-blue-60-transparent-30: "rgba(2, 123, 200, 0.3)"
+  palette-blue-60-transparent-40: "rgba(2, 123, 200, 0.4)"
+  palette-blue-60-transparent-50: "rgba(2, 123, 200, 0.5)"
+  palette-blue-60-transparent-60: "rgba(2, 123, 200, 0.6)"
+  palette-blue-60-transparent-70: "rgba(2, 123, 200, 0.7)"
+  palette-blue-60-transparent-80: "rgba(2, 123, 200, 0.8)"
+  palette-blue-60-transparent-90: "rgba(2, 123, 200, 0.9)"
+  palette-blue-60-transparent-100: "rgba(2, 123, 200, 1)"
```
Comment (serifluous) [inline packages/paste-design-tokens/tokens/aliases/color-palette.yml]: With the `blue-60` change, the rgb value needs to change to 2, 122, 197

Comment (TheSisb) [reply packages/paste-design-tokens/tokens/aliases/color-palette.yml]: Good catch!

File: packages/paste-design-tokens/tokens/themes/sendgrid/global/background-color.yml [hunk 2]
```diff
@@ -43,10 +43,10 @@ props:
 
   # primary background
   color-background-primary-darker:
-    value: "{!palette-blue-70}"
+    value: "{!palette-blue-90}"
```
Comment (serifluous) [inline packages/paste-design-tokens/tokens/themes/sendgrid/global/background-color.yml]: This should be blue-80

File: packages/paste-design-tokens/tokens/themes/sendgrid/global/background-color.yml [hunk 3]
```diff
@@ -43,10 +43,10 @@ props:
...
     comment: Darker background for primary actions or highlights
   color-background-primary-dark:
-    value: "{!palette-blue-60}"
+    value: "{!palette-blue-80}"
```
Comment (serifluous) [inline packages/paste-design-tokens/tokens/themes/sendgrid/global/background-color.yml]: This should be blue-70

File: packages/paste-design-tokens/tokens/themes/sendgrid/global/text-color.yml [hunk 4]
```diff
@@ -31,10 +31,10 @@ props:
     value: "{!palette-blue-40}"
     comment: Light shade of link text to be used in interactions
   color-text-link-dark:
-    value: "{!palette-blue-70}"
+    value: "{!palette-blue-80}"
```
Comment (serifluous) [inline packages/paste-design-tokens/tokens/themes/sendgrid/global/text-color.yml]: This should be blue-70

File: packages/paste-design-tokens/tokens/themes/sendgrid/global/text-color.yml [hunk 5]
```diff
@@ -31,10 +31,10 @@ props:
...
     comment: Dark shade of link text to be used in interactions
   color-text-link-darker:
-    value: "{!palette-blue-80}"
+    value: "{!palette-blue-90}"
```
Comment (serifluous) [inline packages/paste-design-tokens/tokens/themes/sendgrid/global/text-color.yml]: This should be blue-80

**Pull request merged**